Makefile

```diff
@@ -12,32 +12,31 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-DOCKER=1
-
 .PHONY: build build-controller build-importer \
 		docker docker-controller docker-cloner docker-importer \
 		test test-functional test-unit \
 		publish \
 		vet \
 		format \
 		manifests \
-		goveralls
-
-all: docker
+		goveralls \
+		release-description
 
-clean:
+DOCKER=1
 ifeq (${DOCKER}, 1)
-	./hack/build/in-docker "./hack/build/build-go.sh clean; rm -rf bin/* _out/* manifests/generated/* .coverprofile"
+DO=./hack/build/in-docker
```

Comment:
Does make allow for indentation here (and below after else)?

Reply:
`DO` must be declared as a make variable, so it must be left justified.  Indenting it would cause to to be declared as a shell var in a sub shell I think.  This means it wouldn't be accessible to the recipes since they are all executed within their own sub shells as well.